src/applications/ivc-champva/10-10D/tests/10-10D.cypress.spec.js

```diff
@@ -6,32 +6,246 @@ import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-test
 import formConfig from '../config/form';
 import manifest from '../manifest.json';
 
+import {
+  selectYesNoWebComponent,
+  selectRadioWebComponent,
+  fillTextWebComponent,
+  fillFullNameWebComponentPattern,
+  reviewAndSubmitPageFlow,
+  fillDateWebComponentPattern,
+  fillAddressWebComponentPattern,
+} from '../../shared/tests/helpers';
+
+// disable custom scroll-n-focus to minimize interference with input-fills
+formConfig.useCustomScrollAndFocus = false;
+
 const testConfig = createTestConfig(
   {
     dataPrefix: 'data',
-
-    dataDir: path.join(__dirname, 'data'),
-
+    dataDir: path.join(__dirname, 'e2e', 'fixtures', 'data'),
     // Rename and modify the test data as needed.
     dataSets: ['test-data'],
-
+    arrayPages: [{ arrayPath: 'applicants', regex: /.*applicant.*/g }],
     pageHooks: {
       introduction: ({ afterHook }) => {
         afterHook(() => {
-          cy.get('va-button[text*="start"]')
-            .last()
+          cy.findAllByText(/start/i, { selector: 'a' })
+            .first()
             .click();
         });
       },
+      'sponsor-information': ({ afterHook }) => {
+        cy.injectAxeThenAxeCheck();
+        afterHook(() => {
+          cy.get('@testData').then(data => {
+            fillFullNameWebComponentPattern(
+              'veteransFullName',
+              data.veteransFullName,
+            );
+            fillDateWebComponentPattern('sponsorDOB', data.sponsorDOB);
+            cy.axeCheck();
+            cy.findByText(/continue/i, { selector: 'button' }).click();
+          });
+        });
+      },
```

Comment:
Since lots of the screens in this form use web components, have to explicitly tell Cypress how to fill those pages out. This is one of several examples of that.